app/views/examples/translated/index.njk

```diff
@@ -36,6 +36,21 @@
   {{ govukBackLink({
     "href": "/"
   }) }}
+
+  {% call govukNotificationBanner({
```

Comment:
Think we should go with the **Full page examples** approach instead?
https://github.com/alphagov/govuk-frontend/blob/main/app/views/full-page-examples/index.njk#L17

Don't think [Notification Banner](https://design-system.service.gov.uk/components/notification-banner/) fits the purpose

>Use a notification banner to tell the user about something they need to know about, but that’s not directly related to the page content.

>If the information is directly relevant to the thing the user is doing on that page, put the information in the main page content instead. Use [inset text](https://design-system.service.gov.uk/components/inset-text/) or [warning text](https://design-system.service.gov.uk/components/warning-text/) if it needs to stand out.

Reply:
Something plain and simple? @claireashworth

```html
<h1 class="govuk-heading-xl">
  Translated examples
</h1>

<p class="govuk-body-l">These examples are designed to test how patterns and components can be translated.</p>

<p class="govuk-body">Please do not use these translations directly in your application.</p>
```

<img width="628" alt="Image showing the suggested text" src="https://user-images.githubusercontent.com/415517/197005901-0f3076da-3cdf-4139-a17a-11cdbd3175d0.png">

Reply:
If possible, I would swap the second 2 lines to make extra sure the user sees the warning. 
And how about this:

'These examples only illustrate how our components display translated text.'